lib/scsi_defs.h

```diff
@@ -90,5 +95,63 @@ namespace scsi_defs {
     ControlByte control_byte;
   } ABSL_ATTRIBUTE_PACKED; static_assert(sizeof(TestUnitReadyCommand) == kTestUnitReadyCommandSize);
 
+  // SCSI Reference Manual Table 76 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct PersistentReserveInCommand {
+    OpCode op_code = OpCode::kPersistentReserveIn;
+    uint8_t reserved_1 : 3;
+    uint8_t service_action : 5;
+    uint64_t reserved_2 : 40;
+    uint16_t allocation_length; // Big endian
```

Comment:
Are there any multibyte integers in SCSI that *aren't* big-endian? If not, we probably don't need to annotate individual fields.

Reply:
yeah you're right. all of the multi-byte fields are big-endian. There was a comment on an earlier PR about annotating the endianess of fields, but since they're all big-endian, we can jsut have one comment somewhere